README.md

```diff
@@ -1,3 +1,4 @@
+[![Go Report Card](https://goreportcard.com/badge/github.com/nerdalize/nerd)](https://goreportcard.com/report/github.com/nerdalize/nerd)
```

Comment:
Nice! Didn't know this service. Might move the position around in the README update branch